download_air_quality_v2.py

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)